tests/testsuite/package.rs

```diff
@@ -1425,6 +1425,59 @@ edition = "2021"
     );
 }
 
+#[cargo_test]
+fn dirty_file_outside_pkg_root_inside_submodule() {
+    if !symlink_supported() {
```

Comment:
Is there a reason this test was added in this commit?

Reply:
Oops,. to make the suggestions, I was looking at all changes, not a commit, and forgot.